src/config/firebaseConfig.js

```diff
@@ -2,11 +2,14 @@ import firebase from 'firebase/compat/app';
 import 'firebase/compat/auth';
 import { IS_CANARY, IS_PROD } from ".";
 
+const PROD_FIREBASE_API_KEY = process.env.REACT_APP_PROD_FIREBASE_API_KEY
+const LOCAL_FIREBASE_API_KEY = process.env.REACT_APP_LOCAL_FIREBASE_API_KEY
```

Comment:
just a comment: you could just have one variable REACT_APP_FIREBASE_API_KEY that has different values on PROD, CANARY, DEV, LOCAL, which I think that would be simpler and cleaner.  Then we wouldn't have to have a different set of environment variables in each environment.